addons/web_editor/static/src/js/widgets/widgets.js

```diff
@@ -1064,7 +1066,7 @@ var MediaDialog = Dialog.extend({
             }
             var media = self.active.media;
 
-            self.final_data = [media, self.old_media];
+            self.final_data = [media, self.old_media, self.active.images];
```

Comment:
Something is wrong here... why is this part of the 'save' operation and not the 'upload' operation? In fact https://github.com/odoo/odoo/pull/23188/files#diff-1d46d34bc69ceef378d60f3fe61e4bf5R397 seems wrong.

Reply:
I can edit to make it clearer.
On the other hand, the final behavior is normal. We consider that when we add an image in the html, we upload it in the content. It is therefore necessary to add in the attachments the image used.